apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/flavored.gradle

```diff
@@ -0,0 +1,60 @@
+buildscript {
+  repositories {
+    jcenter()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:2.3.0-beta3'
+  }
+}
+
+plugins {
+  id 'com.android.application'
+  id 'com.apollographql.android'
+}
+
+android {
+  compileSdkVersion 25
+  buildToolsVersion '25.0.2'
```

Comment:
Not sure that property will be picked up, as this is a test fixture and I guess gradle plugin test runs it in isolated env from our main gradle.properties file, but I can be wrong.